evennia/contrib/worldloader/README.md

```diff
@@ -0,0 +1,126 @@
+# Intro
+It is a tool for Evennia(http://github.com/evennia/evennia) which can load game world from CSV files.
+
+
+## Installation
+You need to install Evennia and create your game first. Evennia's wiki is here: http://github.com/evennia/evennia/wiki.
+
+Copy the folder ```/example_tutorial_world/worlddata``` to your game folder. The directory should look like
+
+```
+game
+  |
+  ----- commands  
+    |
+    --- server
+    |
+    --- ...
+    |
+    --- worlddata
+```
+
+Add these at the end of the ```/server/conf/settings.py```:
+```
+from worlddata import world_settings
+INSTALLED_APPS = INSTALLED_APPS + (world_settings.WORLD_DATA_APP,)
```

Comment:
If the game world is large, there may be a series of tables and a lot of CSV files. So I think it's better to load them into db and use a new app to hold db models.